examples/architext/architext_test.py

```diff
@@ -0,0 +1,178 @@
+from cheese.api import CHEESE
+from cheese.client.gradio_client import GradioFront, InvalidInputException
+from cheese.data import BatchElement
+from cheese.models import BaseModel
+from cheese.pipeline.write_only import WriteOnlyPipeline
+from dataclasses import dataclass
+from examples.architext.architext_util import prompt_to_layout
+
+import gradio as gr
+import json
+import pickle
+import time
+import torch
+from transformers import AutoTokenizer, AutoModelForCausalLM
+from typing import Dict
+
+
+@dataclass
+class ArchitextBatchElement(BatchElement):
+    prompt : str = ""
+    creativity : str = None # Low, Medium, High
+    result : Dict = None  # { image: PIL.Image.Image, layout: str, layout_after_removed: str }
```

Comment:
While not an immediate concern: If possible, I'd recommend finding some way to get the user the image without explicitly sending it to them. Would it be possible to take model outputted text and convert it into an image client side? If so, it might save on network traffic substantially.

Reply:
Serving text would work well for architext given that our image is just a vector and we can easily draw it on a canvas. However, I don't think that rendering text as an image on the client side is possible without rolling a custom frontend. It appears gradio serves images as a base64 encoded .png and a typical architext generation is around ~10kb. This should be manageable for architext, but generally, this may become a concern if one wants to use an image model in an online flow. In this case one would want to upload the generated image to a cdn and serve that link instead.